PWGLF/TableProducer/lambdakzerobuilder.cxx

```diff
@@ -148,13 +160,225 @@ struct lambdakzeroBuilder {
+        v0dataLink(-1);
+        continue;
+      }
+      registry.fill(HIST("hGoodIndices"), 2.5);
+      if (fabs(V0.posTrack_as<MyTracks>().dcaXY()) < dcapostopv) {
+        MY_DEBUG_MSG(isK0SfromLc, LOG(info) << "posTrack " << labelPos << " has dcaXY " << V0.posTrack_as<MyTracks>().dcaXY() << " , cut at " << dcanegtopv);
+        v0dataLink(-1);
+        continue;
+      }
+      if (fabs(V0.negTrack_as<MyTracks>().dcaXY()) < dcanegtopv) {
+        MY_DEBUG_MSG(isK0SfromLc, LOG(info) << "negTrack " << labelNeg << " has dcaXY " << V0.negTrack_as<MyTracks>().dcaXY() << " , cut at " << dcanegtopv);
+        v0dataLink(-1);
+        continue;
+      }
+      MY_DEBUG_MSG(isK0SfromLc, LOG(info) << "Filling good indices: posTrack --> " << labelPos << ", negTrack --> " << labelNeg);
+      registry.fill(HIST("hGoodIndices"), 3.5);
+
+      // Candidate building part
+      std::array<float, 3> pos = {0.};
+      std::array<float, 3> pvec0 = {0.};
+      std::array<float, 3> pvec1 = {0.};
+
+#ifdef MY_DEBUG
+      auto labelPos = V0.posTrack_as<MyTracks>().mcParticleId();
+      auto labelNeg = V0.negTrack_as<MyTracks>().mcParticleId();
+      bool isK0SfromLc = isK0SfromLcFunc(labelPos, labelNeg, v_labelK0Spos, v_labelK0Sneg);
+#endif
+
+      MY_DEBUG_MSG(isK0SfromLc, LOG(info) << "labelPos = " << labelPos << ", labelNeg = " << labelNeg);
+
+      registry.fill(HIST("hV0Candidate"), 0.5);
+
+      auto pTrack = getTrackParCov(V0.posTrack_as<MyTracks>());
+      auto nTrack = getTrackParCov(V0.negTrack_as<MyTracks>());
+
+      // Require collision-ID
+      if (V0.posTrack_as<MyTracks>().collisionId() != V0.negTrack_as<MyTracks>().collisionId() && rejDiffCollTracks) {
+        v0dataLink(-1);
+        continue;
+      }
+
+      // Act on copies for minimization
+      auto pTrackCopy = o2::track::TrackParCov(pTrack);
+      auto nTrackCopy = o2::track::TrackParCov(nTrack);
+
+      //---/---/---/
+      // Move close to minima
+      int nCand = fitter.process(pTrackCopy, nTrackCopy);
+      if (nCand == 0) {
+        v0dataLink(-1);
+        continue;
+      }
+
+      double finalXpos = fitter.getTrack(0).getX();
+      double finalXneg = fitter.getTrack(1).getX();
+
+      // Rotate to desired alpha
+      pTrack.rotateParam(fitter.getTrack(0).getAlpha());
+      nTrack.rotateParam(fitter.getTrack(1).getAlpha());
+
+      // Retry closer to minimum with material corrections
+      o2::base::Propagator::MatCorrType matCorr = o2::base::Propagator::MatCorrType::USEMatCorrNONE;
+      if (useMatCorrType == 1)
+        matCorr = o2::base::Propagator::MatCorrType::USEMatCorrTGeo;
+      if (useMatCorrType == 2)
+        matCorr = o2::base::Propagator::MatCorrType::USEMatCorrLUT;
+
+      o2::base::Propagator::Instance()->propagateToX(pTrack, finalXpos, d_bz, 0.85f, 2.0f, matCorr);
```

Comment:
Can these `0.85f` and `2.0f` be defined as `constexpr` variables to have a definition?